### PR TITLE
COMMERCE-9130 Only show active shipping method options

### PR DESCRIPTION
**Pull request by dsitu**
https://issues.liferay.com/browse/COMMERCE-9130

### CODE REVIEW

**Comment by dsitu** [comment]
ci:test:sf

**Comment by dsitu** [comment]
ci:test:relevant

**Comment by dsitu** [comment]
ci:test:relevant

**Review by luca-pellizzon — APPROVED** [review]

**Comment by luca-pellizzon** [comment]
ci:test:commerce

**Comment by dsitu** [comment]
ci:test:sf

**Comment by dsitu** [comment]
ci:test:relevant

**Comment by dsitu** [comment]
@luca-pellizzon added 2 commits to fix null pointer error.

**Comment by dsitu** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-account-web/src/main/java/com/liferay/commerce/account/web/internal/portlet/action/EditAccountEntryCommerceShippingOptionMVCActionCommand.java [hunk 1]
```diff
@@ -109,6 +109,10 @@ protected void doProcessAction(
 			return null;
 		}
 
+		if (commerceShippingFixedOption == null) {
```
Comment (luca-pellizzon) [inline modules/apps/commerce/commerce-account-web/src/main/java/com/liferay/commerce/account/web/internal/portlet/action/EditAccountEntryCommerceShippingOptionMVCActionCommand.java:112]: can't we merge this with the previous IF statement?
if (commerceShippingFixedOption == null) {
 if(commerceShippingOptionAccountEntryRel != null){
   ......deleteCommerceShippingOptionAccountEntryRel....
 }
return null;
}

**Comment by dsitu** [comment]
ci:test:sf

**Comment by dsitu** [comment]
ci:test:relevant

**Review by luca-pellizzon — APPROVED** [review]

**Comment by dsitu** [comment]
Closing, commits were added to functional test PR and merged https://github.com/liferay-commerce/liferay-portal/pull/3066.

**Pull request closed without merging**